portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,58 @@
+         working remotely from Mexico. Here's a picture
+         of one of my dogs, he's name is Remmy! </p><br>
+      <img src = "/myImages/remmy.jpeg" alt = "Remmy" width = "200" height = "300" /> <br>
+
+      <h2>My Background</h2>
+      <p> I have the oportunity to study abroad bacause I live in one of the borders 
+        with the US. When I was little my ideas were influeced by a lot of people 
+        due to my country's environment. When I turned 8 years old, I started to 
+        practice Taekwondo, which was a turning point in my life. That environment
+        showed my different perspectives of life, specially when it came to gender 
+        equality. </p><br>
+
+      <h2> Revelant Projects </h2>
+        <li> [High School] Analyzed lake water from my town's Central's park. The purpose
+            was to research the living beings in the water that were turning the water green
+            as time passed. </li>
+        <li> [UTEP] Worked with linked lists and sorting concepts to translate RNA sequences 
+            to Amino Acid in one of the lab assignments. </li>
+        <li> [UTEP] Worked with stacks and queues by using arithmetic expressions stored in
+            binary trees using postfix notation.</li><br>
+
+      <h2> Contact Information </h2>
+      <a href = "http://linkedin.com/in/diana-valles-57b2571a3"> LinkedIn profile </a><br>
+      <a href = "https://github.com/dianapvalles"> GitHub profile </a><br>
+
+      <div class = "image-thumbnails img">
+          <h2><b>Photo Collection</b></h2>
+        <a href="/myImages/breeze.jpeg"><img src="myImages/breeze.jpeg" alt = "Chicago's Navy Pier"/></a>
+        <a href="/myImages/clouds.jpeg"><img src="myImages/clouds.jpeg" alt = "Landcape"/></a>
+        <a href="/myImages/concert.jpeg"><img src="myImages/concert.jpeg" alt = "EP concert"/></a>
+        <a href="/myImages/dogs.jpeg"><img src="myImages/dogs.jpeg" alt = "2/3 dogs"/></a>
+        <a href="/myImages/flowers.jpeg"><img src="myImages/flowers.jpeg" alt = "Flowers"/></a>
+      </div>
+
+      <h3> Why do I enjoy photographing? </h3>
+        <p> Photography adds too much value to our lives. I like to re-live moments by just looking at
+          old photos. It is a way to share our life experiences with others. It also lets you explore 
+          your creative side. </p><br>
+      
+      <h2> Hidden Talent? </h3>
+          <p> I believe I am good at sports. I particularly spent several years practicing
+              Taekwondo and got my black belt on September 2015. I coached multiple kids in
+              tournaments and gave some classes too. </p><br>
+      <img src = "/myImages/tournament.jpeg" alt = "tournament" width = "300" height = "200" /> <br>
+
+      <p>Click here to get a random quote from my favorite 
+         TV shows or fact about me:</p>
+      <button onclick="addRandomGreeting()">Click</button>
       <div id="greeting-container"></div>
-    </div>
+    
```

Comment:
removing `</div>` means that the `<div id="content">` at the top isn't closed now

Reply:
You can use an online html validator to find all of these https://www.freeformatter.com/html-validator.html

Reply:
Thank you for the resource and review!